### ISSUE TITLE
Clarify sharing is to one conversation with N people, not N 1:1 conversations

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#5057]
When you use the "share to Zulip" feature and select PMs, we show you a UI to select one or more users from the list of all the users.

In some other apps, for example Facebook Messenger, when you share to that app there's a pretty similar-looking UI and what it means is that you're choosing a list of conversations to share the photo/file/whatever into. Any group conversations you have will also appear in that list, but mostly it's a list of other users, each of them standing for your 1:1 conversation with that user.

But what we actually mean by that UI is that you're going to share into a single conversation, and you're choosing the set of other users in it.

@AlaaElden98 encountered this behavior while working on #5044, and was quite confused ([chat thread](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/share.20to.20Zulip/near/1266275)) because (IIUC) he was sure the UI was indicating it would share into a set of 1:1 conversations, and then he wasn't seeing the messages in the expected place.

I think the behavior we have is probably the one we want: Zulip makes many-person discussions work better than in other apps, and in Zulip the most natural way to share something with N people is going to be in a stream they're in, followed by in a group-PM thread with them, not in N separate 1:1 conversations with them.

But we should make it clear in the UI that that's what's going on.

**Comment by Nadrieril** [comment]
A way of making this clearer would be that in the "private message" tab of the share dialogue, we show the list of existing private conversations. Then if a user wants to share to N people they can create a new group PM and then share to that. To make this easy the "new PM"/"new group PM" buttons would be available directly from the share dialogue.
I also would really like this because I always share to an existing conversation and it's painful to have to select recipients manually each time.

**Comment by alya** [comment]
Yeah, having the "Choose recipients" button take you to the private messages tab makes sense to me. @AkashDhiman @gnprice does that sound right? How complex is this change?
